smart-contracts/test/Lock/erc20.js

```diff
@@ -147,7 +147,9 @@ contract('Lock / erc20', accounts => {
       await token.approve(lock.address, -1)
```

Comment:
CAn you remind me what the `-1` is in the approve call?

Reply:
it's shorthand for MAX_UINT

Reply:
ahh, of couse,thx